src/mom5/ocean_core/ocean_util.F90

```diff
@@ -845,7 +845,8 @@ subroutine diagnose_3d_mask(Time, mask, id_name, data, nk_lim, use_mask, abs_max
 
     logical :: use_mask_, used
     integer :: nk_lim_
-    real, dimension(isd:ied,jsd:jed,nk) :: threshold_mask
+!    real, dimension(isd:ied,jsd:jed,nk) :: threshold_mask
```

Comment:
Should remove the previous (commented) definition

Reply:
Done